climada/engine/unsequa/calc_cost_benefit.py

```diff
@@ -127,7 +130,11 @@ def __init__(
 
 
 
-    def uncertainty(self, unc_data, pool=None, **cost_benefit_kwargs):
+    def uncertainty(self,
```

Comment:
Yes, but that is the problem. The names of the data are very specific to Impact and CostBenefit. I do not see how this could be resolved without confusion. Note also that the computation step in the end is specific... Suggestions are welcome!

Reply:
After discussion: No fix in sight